### PR TITLE
Fix Developer Guide for some formatting issues

### PR DESCRIPTION
**Pull request by Hikoya**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -581,20 +581,20 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 | `*`      | impatient user            | get a visualization for any loading times                                                                                    | know how long I need to wait                                                                                                                           | No                                                |
-| `* *`    | user                      | search for past reviews by substring                                                                                         | see places I’ve been to before                                                                                                                         | No (Exact string match as of now by Review name)  |
+| `* *`    | user                      | search for past reviews by substring                                                                                         | see places I’ve been to before                                                                                                                         | No (Exact string match as of now by name)         |
```
Comment (MinHeinA) [inline docs/DeveloperGuide.md:597]: Stall name would sound better?

**Review by MinHeinA — COMMENTED** [review]
Minor concern.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Hikoya) [reply docs/DeveloperGuide.md]: Fixed in latest commit.

**Review by MinHeinA — APPROVED** [review]
LGTM!

**Pull request merged**